extensions/wikia/TemplateClassification/scripts/TemplateClassificationModal.js

```diff
@@ -141,8 +147,13 @@ function ($, mw, loader, nirvana, tracker, labeling) {
 			});
 		});
 
+		$(w).bind('keypress', {modalInstance: modalInstance}, submitFormOnEnterKeyPress);
+
 		/* Show the modal */
 		modalInstance.show();
+
+		// Make sure that focus is in the right place
+		$('#template-classification-' + mw.html.escape($preselectedType.val())).focus();
 	}
 
 	function processSave(modalInstance) {
```

Comment:
Can we distinguish tracking submit via keyboard from regular click 
https://github.com/Wikia/app/blob/CE-2963/extensions/wikia/TemplateClassification/scripts/TemplateClassificationModal.js#L170
like we do on closing modal?